app/lib/shared/versions.dart

```diff
@@ -17,6 +17,18 @@ import 'utils.dart' show isNewer;
 /// a future dates are used.
 final RegExp runtimeVersionPattern = RegExp(r'^\d{4}\.\d{2}\.\d{2}$');
 
+/// The list (and priority order) of runtimeVersions where version-specific data
+/// is accepted from.
+///
+/// Make sure that at least two versions are kept here as the next candidates
+/// when the version switch happens.
+const acceptedRuntimeVersions = <String>[
+  '2020.05.06',
+  '2020.05.03',
+  '2020.04.22',
+  '2020.04.07',
```

Comment:
Maybe recommend that we don't let's this grow larger than 7 entries? or what do you think..

Reply:
I'll write a test for it, limiting it to 5.